File: Daily Challenge/C++/Flowchart Logic - 060 5.11.2025.cpp
#include <iostream>
using namespace std;

int main() {
    int N, D1, D2;
    cin >> N >> D1 >> D2;
    bool ok = true;
    while (N > 0) {
        int d = N % 10;
        if (d < D1 || d > D2) {
            cout << "NO" << endl;
            ok = false;
            break;
        }
        N /= 10;
    }
    if (ok) cout << "YES" << endl;
}

File: Daily Challenge/Java/Flowchart Logic - 060 5.11.2025.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int D1 = sc.nextInt();
        int D2 = sc.nextInt();
        boolean ok = true;
        while (N > 0) {
            int d = N % 10;
            if (d < D1 || d > D2) {
                System.out.println("NO");
                ok = false;
                break;
            }
            N /= 10;
        }
        if (ok)
            System.out.println("YES");
    }
}